libmachine/provision/os_release.go

```diff
@@ -69,7 +69,7 @@ func (osr *OsRelease) ParseOsRelease(osReleaseContents []byte) error {
 	for scanner.Scan() {
 		key, val, err := parseLine(scanner.Text())
 		if err != nil {
-			return err
+			log.Warn("Warning: got an invalid line error parsing /etc/os-release: %s", err)
```

Comment:
Shouldn't we also skip this line since it's invalid? Probably just `continue` after the `log.Warn`